LanguageExt.Core/Prelude_Union.cs

```diff
@@ -0,0 +1,366 @@
+﻿namespace LanguageExt
+{
+	using System;
+
+	public static partial class Prelude
+	{
+		public static IUnion<B> Bind<A, B>(this IUnion<A> a, Func<A, IUnion<B>> func) =>
```

Comment:
Could we be more consistent with Where here? Seems that `a` and `func` are really `source` and `predicate` like line 16.